src/store/action.ts

```diff
@@ -0,0 +1,15 @@
+import { createAction } from '@reduxjs/toolkit';
+import { City, Offer } from '../types/offers';
+
+export const Action = {
+  CHANGE_CITY: 'CHANGE_CITY',
+  SELECT_OFFERS: 'SELECT_OFFERS'
+};
+
+export const changeCity = createAction(Action.CHANGE_CITY, (selectedCity: City) => ({
+  payload: selectedCity
+}));
+
+export const selectOffers = createAction(Action.SELECT_OFFERS, (currentOffers: Offer[]) => ({
+  payload: currentOffers
+}));
```

Comment:
tbh, я не запускал это в твоём репозитории, но `redux-toolkit` предоставляет  сокращённый синтаксис для `createAction`, где можно обойтись без необходимости явно задавать строки типов действий (**action types**) и конструкцию второго аргумента (**payload creator**), если она проста.

```suggestion
export const changeCity = createAction<City>('CHANGE_CITY');
export const selectOffers = createAction<Offer[]>('SELECT_OFFERS');
```